modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase

```diff
@@ -31,6 +31,10 @@ definition {
 
 			Navigator.gotoPage(pageName = "Frontend Data Set Test Page");
 		}
+
+		task ("And Given: site URL localized to Spanish //localhost:8080/es/*") {
+			Navigator.openSpecificURL(url = "http://localhost:8080/es/frontend-data-set-test-page/");
```

Comment:
You can change the localhost to use a dynamic address, CI does not use port 8080, so the test is breaking.

```
var portalURL = PropsUtil.get("portal.url");

Navigator.openSpecificURLvalue1 = "${portalURL}es/frontend-data-set-test-page/");
```

Reply:
And can you move this task for inside the test?

Reply:
okay!